packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -66,11 +66,13 @@ const propTypes = {
    * Callback function that is called when a selectable cell is selected. Parameters:
    * @param {string} rowId rowId
    * @param {string} columnId columnId
+   * @param {string} sectionId sectionId
```

Comment:
We will want to return an object with named parameters for the section id, row id, and column id.